index.js

```diff
@@ -138,3 +138,54 @@ newSightingForm.addEventListener('submit', function (event) {
 /************************** EVENTS PART 2 JS MINI CHALLENGE ******************************/
 
 
+//**********Deliverable 1: Like an animal sighting**********
+
+const animSightings = document.querySelector('ul#animals')
+
+animSightings.addEventListener('click', function (event){
+if (event.target.matches('button.like-button')){
+    const likesDisplay = event.target.previousElementSibling
+    const animLikes = parseInt(likesDisplay.textContent) + 1
+    likesDisplay.textContent = `${animLikes} Likes`
+}
+
+//**********Deliverable 2: Delete an animal sighting**********
+
+else if (event.target.matches('button.delete-button')){
+    const animals = event.target.closest('li')
+    animals.remove()
+}
+
+//**********Deliverable 3: View the update animal sighting description form**********
+
+else if (event.target.matches('button.toggle-update-form-button')){
+    const li = event.target.closest('li')
+    const showForm = li.querySelector('form')
+
+    if(showForm.style.display === 'block'){
+        showForm.style.display = 'none'
+    }
+    else{
+        showForm.style.display = 'block'
+        }
+    }
+})
```

Comment:
Be careful with indentation here. 

Nice! You have an event listener on the entire animals UL element, and based on the class names you are delegating out different logic. Great. First step is getting everything up and running. Awesome. Next, we can refactor for readability and to make our code more reusable.

```
/***** DELIVERABLE 1, 2, & 3 part 1 *****/

const animalsUL = document.querySelector('ul#animals')

animalsUL.addEventListener("click", function(event){
    if (event.target.matches('button.like-button')){
        handleLike(event)
    } else if (event.target.matches('button.delete-button')){
        handleDelete(event)
    } else if (event.target.matches('button.toggle-update-form-button')){
        handleToggleUpdate(event)
    }
})

/***** DELIVERABLE 1 part 2 *****/
function handleLike(e){
    const likesDisplayPTag = e.target.previousElementSibling
    const currentTextContent = likesDisplayPTag.innerText
    let numberOfLikes = parseInt(currentTextContent.split(" ")[0])
    numberOfLikes += 1
    const newTextContent = `${numberOfLikes} Likes`
    likesDisplayPTag.innerText = newTextContent
}

/***** DELIVERABLE 2 part 2 *****/
function handleDelete(e){
    const parentLI = e.target.parentElement
    parentLI.remove()
}

/***** DELIVERABLE 3 part 2 *****/
function handleToggleUpdate(e){
    const updateForm = e.target.nextElementSibling
    toggleHidden(updateForm)
}

/***** DELIVERABLE 3 part 3 *****/
function toggleHidden(element){
    element.style.display === "none" ? element.style.display = "block" : element.style.display = "none"
}
```